src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/EnvironmentHelper.cs

```diff
@@ -33,15 +34,16 @@ public sealed class ApplicationConfig
 			public uint   system_property_count;
 			public uint   number_of_assemblies_in_apk;
 			public uint   bundled_assembly_name_width;
-			public uint   number_of_assembly_blobs;
+			public uint   number_of_assembly_store_files;
```

Comment:
Why rename "blobs" to "files"?  Clarity?  Other?

Reply:
I thought we wanted to drop the word `blob` usage. And I didn't want to use `stores` because it didn't sound right to me and since they are files... :)